src/ui/pages/Settings/Settings.tsx

```diff
@@ -340,6 +343,44 @@ function ToggleSettingLine({
   );
 }
 
+function ClearPendingTransactionsLine() {
+  const toastRef = useRef<PopoverToastHandle>(null);
+  const { mutate: clearPendingTransactions, ...mutation } = useMutation({
+    mutationFn: async () => {
+      await new Promise((r) => setTimeout(r, 500)); // artificial delay
+      toastRef.current?.removeToast();
+      return walletPort.request('clearPendingTransactions');
+    },
+    onSuccess: () => {
+      toastRef.current?.showToast();
+    },
+  });
+  return (
+    <>
+      <PopoverToast ref={toastRef}>Pending transactions cleared</PopoverToast>
+
+      <Frame
+        as={UnstyledButton}
+        interactiveStyles={true}
+        onClick={() => clearPendingTransactions()}
+        disabled={mutation.isLoading}
+        style={{ padding: 20 }}
+      >
+        <Media
+          image={null}
+          text={<UIText kind="body/accent">Clear Pending Transactions</UIText>}
+          vGap={0}
+          detailText={
+            <UIText kind="small/regular" color="var(--neutral-500)">
+              This can fix stuck transactions
```

Comment:
I'm not sure what I feel about this description. Not sure that this actually FIXES anything :)